docs/src/main/paradox/docs/releases/v1.7-to-v1.8-migration.md

```diff
@@ -0,0 +1,453 @@
+# v1.7 To v1.8 Migration
+
+The v1.8 release of Delta introduces PostgreSQL as sole option for the Delta primary event store.
+
+(To have a complete list of improvements, @ref:[see here](./v1.8-release-notes.md)).
+
+These improvements require to replay all existing events, moving them from Cassandra to PostgreSQL, and to reindex all
+data to elasticsearch/Blazegraph.
+
+The following steps describe how to perform the migration. At the end, Delta v1.8 will be running using PostgreSQL as
+primary store.
+
+Depending on the number and the complexity of existing resources and schemas in the system, the migration may take
+several hours to complete.
+
+The migration itself can be run in three ways:
+
+* Switching off your Delta 1.7-base Nexus deployment, and re-starting it with Delta 1.8. Then re-uploading all the data
```

Comment:
It could be nice to introduce a diagram with the double run setup and make it evolve in the different steps